.travis.yml

```diff
@@ -15,8 +15,16 @@ sudo: true
 git:
   depth: false
 before_install:
-- openssl aes-256-cbc -K $encrypted_342342ee5e49_key -iv $encrypted_342342ee5e49_iv
-  -in gcp-credentials.json.enc -out gcp-credentials.json -d
+- |
+  if [ "$TRAVIS_PULL_REQUEST_SLUG" != "pulumi/pulumi" ]; 
+  then 
+    echo "Not decrypting gcp credentials";
+    export COMMUNITY_PR="true"
```

Comment:
do we need to export an environment variable? The tests won't fail if it's a community PR - the tests will skip due to not being able to read PULUMI_ACCESS_TOKEN

Reply:
Some of the tests aren't skipped even if that value isn't set, see here:

https://travis-ci.com/github/pulumi/pulumi/builds/162335371#L3290

I'm happy to set the skip to be if `PULUMI_ACCESS_TOKEN` isn't set if we think that's better?

Reply:
I've switched this to use `PULUMI_ACCESS_TOKEN` instead